pinserver/exports.sh

```diff
@@ -0,0 +1,7 @@
+export APP_PINSERVER_PORT="8096"
```

Comment:
Are you happy for us to change this? Will conflict with Jellyfin unfortunately https://github.com/getumbrel/umbrel-apps/blob/master/jellyfin/umbrel-app.yml

Reply:
do you have an available port? i can update also images to have the same port exposed in the image

Reply:
Looks like port 8097 is free, i can move app on this port.
In next days if ok i will move on this port and update docker images.

Reply:
perfect, thanks